content/blog/2018-07-17-pkginspector.md

```diff
@@ -0,0 +1,112 @@
+---
+slug: "pkginspector"
+title: "What's inside? __pkginspector__ provides helpful tools for inspecting package contents"
+preface: "This post describes a project from rOpenSci unconf18. In the spirit of exploration and experimentation at our unconferences, projects are not necessarily finished products or in scope for rOpenSci packages."
+authors:
+    
+  - name: Sam Albers
+    url: https://github.com/boshek
+  - name: Leonardo Collado-Torres
+    url: https://github.com/lcolladotor
+  - name: Mauro Lepore
+    url: https://github.com/maurolepore
+  - name: Joyce Robbins
+    url: https://github.com/jtr13
+  - name: Noam Ross
+    url: https://github.com/noamross
+  - name: Omayma Said
+    url: https://github.com/OmaymaS   
+date: '2018-07-17'
+categories: blog
+topicid: ####
+tags:
+  - community
+  - r
+  - review
+  - onboarding
+  - package
+  - unconf
+  - unconf18
+---
+
+R packages are widely used in science. Despite this broad usage, the code behind R packages is rarely part of the scientific peer-review process. In the R ecosystem, rOpenSci has been a pioneer in developing a peer-review process for R packages. The goal of [__pkginspector__](https://github.com/ropenscilabs/pkginspector/) is to help that process by providing a means to better understand the internal structure of R packages. It summarizes the relationship among functions within a package, and reports whether or not functions' interfaces are consistent. If you are reviewing an R package (maybe your own) __pkginspector__ is for you (or at least it will be!).
+
+We begun building __pkginspector__ during [unconf18](http://unconf18.ropensci.org/), with support from [rOpenSci](https://ropensci.org/) and guidance from [Noam Ross](https://github.com/noamross). The package focuses on facilitating a few of the many tasks involved in reviewing a package. (For more on rOpenSci's review process, see the blog post: ["Onboarding at rOpenSci: A Year in Reviews"](https://ropensci.org/blog/2016/03/28/software-review/) and the e-book [*rOpenSci Packages: Development, Maintenance, and Peer Review.*](https://ropensci.github.io/dev_guide/)) Specifically, we are building tools to analyze and visualize function dependencies, and to analyze function parameters within a package.
```

Comment:
Can you say what a reviewer needs to do manually if they don't have `pkginspector` to help? Either in intro or to tie this into concluding paragraph